test/gtest/common/test_helpers.h

```diff
@@ -461,5 +461,23 @@ class message_stream {
             } \
         }
 
+/**
+ * Make ucp_dt_iov_t iov[iovcnt] array with pointer elements to original buffer
+ */
+#define UCS_TEST_GET_BUFFER_DT_IOV(_name_iov, _name_iovcnt, _buffer_ptr, _buffer_length, _iovcnt) \
+        ucp_dt_iov_t _name_iov[_iovcnt]; \
+        const size_t _name_iovcnt = _iovcnt; \
+        const size_t _buffer_iov_length = _buffer_length / _name_iovcnt; \
```

Comment:
@yosefe Did you mean - using random length in the code there this macro called? What range would you like for iovcnt?